dags/iop/parser.py

```diff
@@ -75,3 +80,13 @@ def _get_related_article_doi(self, article):
         except AttributeError:
             self.logger.error("No related article dois found", dois=self.dois)
             return
+
+    def _get_arxiv_eprints(self, article):
+        arxivs_value = article.find(
+            "front/article-meta/custom-meta-group/custom-meta/meta-value"
+        )
+        try:
+            pattern = re.compile(r"(arxiv:|v[0-9]$)", flags=re.I)
```

Comment:
I would move this pattern to the constant `ARXIV_PATTERN` or something like that cause, for sure, you will need to reuse it somewhere in the repo. Or, at least, I would make it a property of the spider with a proper name.